### DISCUSSION TITLE
List of all the updates that have been made to the app android

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#108263] [Mobile]
### Select Topic Area

Product Feedback

### Body

It would be useful to have a button in the settings section that when clicked opens a section that shows you every update that the app immediately has with its date.

It could also show some images and videos which would be shown via http calls to see what the new features are.

Like VS Code does for example, which every time there is an update shows you what has been changed and explains how.

Like the `Open Source Libraries` section.

**Comment by HenSquared** [comment]
Hi @Angelk90 ,

I've created an Issue to discuss and track this feedback internally. 

Thanks for the feedback!

**Reply by Angelk90** [reply]
@HenSquared : is there anything new?
